pkg/kubevirt/core/util.go

```diff
@@ -57,10 +60,84 @@ func encodeProviderID(machineID string) string {
 	return fmt.Sprintf("%s/%s", ProviderName, machineID)
 }
 
+func buildNetworks(networkSpecs []api.NetworkSpec) ([]kubevirtv1.Interface, []kubevirtv1.Network, string) {
+	var interfaces []kubevirtv1.Interface
+	var networks []kubevirtv1.Network
+
+	// Determine whether there is a default network
+	hasDefault := false
```

Comment:
Can be deleted once grouped as the above comment suggests.

Reply:
See above.